MANIFEST.in

```diff
@@ -7,5 +6,7 @@ recursive-exclude ci *
 recursive-exclude conda.recipe *
 recursive-exclude examples wout*.nc mercier* jxbout* spec*.sp spec*.sp.* .gitignore
 recursive-exclude tests .gitignore
+recursive-include thirdparty
+recursive-exclude thirdparty .git .gitignore .github/* .appveyor.yml *readthedocs.yml .travis.yml azure-pipelines.yml
```

Comment:
I'm suprised that `thirdparty` is still among the folder behind `recursive-exclude`, is that correct?

Reply:
The command is `recursive-exclude dir pattern1 pattern2 ...`
All the files matching subsequent patterns on the same line should be excluded from the thirdparty folder. 
However, I am not sure about the interaction between lines 9 and 10.

Reply:
Just checked the [packages built](https://github.com/hiddenSymmetries/simsopt/suites/6210409075/artifacts/219589510). It is working as expected.